src/analog/utils/CommandBinaryBuffer.ts

```diff
@@ -1341,6 +1362,30 @@ const deviceParameterConvertersMap = {
         set: ( buffer: ICommandBinaryBuffer, parameter: IParameterNbiotApn ) => {
             buffer.setString(parameter.apn);
         }
+    },
+    [deviceParameters.NBIOT_LED_INDICATION]: {
+        get: ( buffer: ICommandBinaryBuffer ): IParameterNbiotLedIndication => (
```

Comment:
```suggestion
        get: ( buffer: ICommandBinaryBuffer ): IParameterNbiotLedIndication => ({
```

?

a little bit more compact

Reply:
fixed